spec/config/settings_spec.rb

```diff
@@ -21,7 +21,7 @@
   describe ".features" do
     features = settings[:features]
 
-    include_examples expected_value_test, :draft_live_versioning, features, false
+    include_examples expected_value_test, :basic_routing, features, false
```

Comment:
Noticed this was missing so replaced `draft_live_versioning` with `basic_routing`